pkgs/dart_mcp/example/dash_client.dart

```diff
@@ -136,16 +157,23 @@ final class DashClient extends MCPClient with RootsSupport {
 
   /// Handles a function call response from the model.
   Future<String?> _handleFunctionCall(gemini.FunctionCall functionCall) async {
-    await _chatToUser(
-      'It looks like you want to invoke tool ${functionCall.name} with args '
-      '${jsonEncode(functionCall.args)}, is that correct?',
-    );
-    final userResponse = await stdinQueue.next;
-    final wasApproval = await _analyzeSentiment(userResponse);
+    if (auto) {
+      await _chatToUser(
+        'I am going to run the ${functionCall.name} tool with args '
```

Comment:
maybe:

```suggestion
        Running the ${functionCall.name} tool with args '
```

Because it immediately starts running it :)